pandas/core/strings.py

```diff
@@ -1842,23 +1842,45 @@ def extractall(self, pat, flags=0):
         return str_extractall(self._orig, pat, flags=flags)
 
     _shared_docs['find'] = ("""
+    Return the lowest index of the substring.
+
     Return %(side)s indexes in each strings in the Series/Index
     where the substring is fully contained between [start:end].
     Return -1 on failure. Equivalent to standard :meth:`str.%(method)s`.
 
     Parameters
     ----------
     sub : str
-        Substring being searched
+        Substring being searched.
     start : int
-        Left edge index
+        Left edge index.
```

Comment:
Can you mention for both start and end the default behaviour?

Reply:
I havent touched it. It was in that way in documentation

Reply:
Sure, It's Done!

Reply:
@jorisvandenbossche examples to be defined in the `@Appender` in `dict`?

Reply:
is it the right way to do it?
```
@Appender(_shared_docs['find'] %
              dict(side='lowest', method='find',
                   also='rfind : Return highest indexes in each strings')
                   examples='''>>> s = pd.Series([...])
                  <output>
                  >>> s = ...
                  ...
```
and in _shared_docs
```
Examples
-------------
%(examples)s
```

Reply:
@ChiragSehra best to do it with little stubs I think.

Something like

```python
_find_examples = """
>>> s = pd.Series([...])
"""

_rfind_examples = """
...
"""
```

Then in the `Appender` do `examples=find_examples`.

Reply:
Alternatively, I think it's fine to *not* do the examples as a variable to be substituted. Having both examples for both `.find` and `.rfind` in the same docstring can be helpful to illustrate the difference.

Reply:
Okay! Will work on it.

Reply:
@TomAugspurger I am not able to validate the changes i made using the `validate_docstrings.py` in `scripts` folder. What should I do now?

Reply:
Can you push your changes so I can take a look?

Reply:
@TomAugspurger Sure, I will push it soon.